mobify/platform-scaffold
# Redesign Preloader

**JIRA**: [WEB-1283](https://mobify.atlassian.net/browse/WEB-1283)

## Changes
- Redesign preloader to match the app shell
- Update brand color to be more accessible (i.e. when used as
background with white text)
- Reverse the background shimmer animation (best practice as prescribed
by our designers)

| Before | After |
| ------- | ------ |
| ![screen shot 2017-03-14 at 3 34 52 pm](https://cloud.githubusercontent.com/assets/734535/23925319/d29077cc-08cb-11e7-96ea-adb91ecf8bab.png) | ![screen shot 2017-03-14 at 3 34 32 pm](https://cloud.githubusercontent.com/assets/734535/23925320/d7b833a2-08cb-11e7-8e01-0721ece483ec.png) |

## Impact

Predicting that Web Page Test will perceive the preloader as a completion of more like `10-20%` instead of `3%` (see below screenshot [from Web Page Test](http://www.webpagetest.org/video/compare.php?tests=170314_GM_16RD-r:1-c:0)). This should improve our page speed index by a bit... not nearly enough, but hopefully combined with upcoming changes, the page speed index will eventually be satisfactory.

Predicted impact:

![screen shot 2017-03-14 at 3 40 22 pm](https://cloud.githubusercontent.com/assets/734535/23925539/e9ecad36-08cc-11e7-8692-5421b5fc002d.png)

Old preloader visual progress:

![screen shot 2017-03-14 at 3 38 27 pm](https://cloud.githubusercontent.com/assets/734535/23925420/4cbb068e-08cc-11e7-982d-100d8f98ad66.png)

> _Notice that the old preloader only ever appeared 3% "complete", but the placeholder app sell was 28% "complete". Hopefully the redesigned preloader will only appear slightly less complete than the placeholdered app._


## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify the design of the Preloader
